src/modules/fingerprint_acl/fp_acl_file.h

```diff
@@ -24,6 +24,10 @@
  *         hasValue: uint8_t, 
  *         value: uint8_t[16]
  *    }
+ *    fcmTok: {
+ *         hasValue: uint8_t, 
+ *         value: uint8_t[255]
```

Comment:
this is definition is incompatible with write_string(write_buffer, user.fcmTok.value.data...

Make it either a null terminated string or a generic buffer with a length. We use the byte hasValue for the PSK and the fingerprint since these values is static length binary values. The fcm token is not a static length binary value. It's is ok to make it optional if we think it's important to distinguish the empty value from the non existsting value (that is probably not the case).